pylib/weather_im/iembot_config.py

```
@@ -285,6 +292,10 @@ def application(environ: dict, start_response: callable):
             except Exception as exp:
                 conn.rollback()
                 flash_message = f"Unable to process request: {exp}"
+            try:
+                reload_bot()
+            except Exception as exp:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)